README.md

```diff
@@ -57,6 +57,21 @@ where
  * `cleanup_interval` (default: 1 hour) is the number of seconds between database cleanups 
  * `metrics_lifetime` (default: 24 hours) is the number of seconds that must pass from creation before each metric can be deleted during cleanup
 
+## Metrics visualisation with Grafana
+
+Metrics stored in the database using `membrane_rtc_engine_timescaledb` can be simply visualized using Grafana.
+To start the dashboard with RTC Engine metrics, pass volume with the content of `priv/grafana/provisioning` to the Grafana docker container.
```

Comment:
How about writing a a brief description of what does the `priv/grafana/provisioning` contain?